uqcsbot/scripts/events.py

```diff
@@ -6,42 +6,55 @@
 from pytz import timezone, utc
 from typing import Tuple, Optional
 from uqcsbot import bot, Command
-from uqcsbot.utils.command_utils import UsageSyntaxException
+from uqcsbot.utils.command_utils import UsageSyntaxException, loading_status
 from uqcsbot.utils.itee_seminar_utils import (get_seminars, HttpException, InvalidFormatException)
 
-CALENDAR_URL = "https://calendar.google.com/calendar/ical/q3n3pce86072n9knt3pt65fhio%40group.calendar.google.com/public/basic.ics"  # noqa
-FILTER_REGEX = re.compile('(full|all|[0-9]+( weeks?)?)')
+CALENDAR_URL = ("https://calendar.google.com/calendar/ical/"
+                "q3n3pce86072n9knt3pt65fhio%40group.calendar.google.com/public/basic.ics")
+FILTER_REGEX = re.compile('full|all|[0-9]+( weeks?)?|jan.*|feb.*|mar.*|apr.*|may.*|jun.*'
```

Comment:
If you're going to be doing your own regex pattern matching, we're going to want to see some tests to confirm it works as intended.

Reply:
Added tests.